libs/langchain/langchain/utilities/steam.py

```diff
@@ -0,0 +1,159 @@
+"""Util that calls Steam-WebAPI."""
+
+from typing import Any
+
+import steamspypi
```

Comment:
these should be conditionally imported

Reply:
Sorry, I'm a bit confused about what you mean by this, do you mean adding a try-catch to the import? I also moved the import to only the function that uses it.

```
def recommended_games(self, steam_id: str) -> dict:
        try:
            import steamspypi
        except ImportError:
            raise ImportError("steamspypi library is not installed.")
```

Reply:
Hi, thanks for reviewing our PR, can you kindly let us know if the error happened in the notebook?